### PR TITLE
feat(screen-share): Create screen-share setting

### PR DESCRIPTION
**Pull request by michalszorad**
Resolves https://emurgo.atlassian.net/browse/YOMO-960

### CODE REVIEW

File: apps/wallet-mobile/src/features/ScreenShare/index.ts
Comment (stackchain) [inline apps/wallet-mobile/src/features/ScreenShare/index.ts:1]: if it's only accessible through settings, move into settings.

Comment (michalszorad) [reply apps/wallet-mobile/src/features/ScreenShare/index.ts:1]: Catalyst QrCode component uses `useAllowScreenshots` imported from here, should we move it anyway?

Comment (stackchain) [reply apps/wallet-mobile/src/features/ScreenShare/index.ts:1]: we need a place for the mobile hooks, and not the wallet hooks, I'd say this is a mobile hook, add to the utils for now.

**Review by stackchain — APPROVED** [review]
lgtm

**Comment by michalszorad** [comment]
Still WIP, needs translations @stackchain

**Comment by michalszorad** [comment]
Reay to be reviewed & merged

**Pull request merged**